paymentsheet/src/main/java/com/stripe/android/link/LinkScreenContent.kt

```diff
@@ -17,22 +18,22 @@ import kotlinx.coroutines.flow.SharedFlow
 @Composable
 internal fun LinkScreenContent(
     viewModel: LinkActivityViewModel,
-    onBackPressed: () -> Unit
+    bottomSheetState: StripeBottomSheetState,
 ) {
     val screenState by viewModel.linkScreenState.collectAsState()
     val appBarState by viewModel.linkAppBarState.collectAsState()
 
     LinkScreenContentBody(
+        bottomSheetState = bottomSheetState,
         screenState = screenState,
         appBarState = appBarState,
         eventReporter = viewModel.eventReporter,
         onVerificationSucceeded = viewModel::onVerificationSucceeded,
         onDismissClicked = viewModel::onDismissVerificationClicked,
-        onBackPressed = onBackPressed,
+        onBackPressed = viewModel::goBack,
```

Comment:
Using the `goBack` method instead of the back dispatcher so that `StripeBottomSheetKeyboardHandler` gets invoked and dismisses the keyboard before back navigation.